pkg/config/control_plane.go

```diff
@@ -195,7 +194,7 @@ type InsightCollectorApplication struct {
 }
 
 type InsightCollectorDeployment struct {
-	Enabled bool `json:"enabled"`
+	Enabled bool `json:"enabled" default:"true"`
```

Comment:
should be `*bool` if its default value is true
like this: https://github.com/pipe-cd/pipecd/blob/master/pkg/config/application_kubernetes.go#L96

Reply:
Oops. Nice catch!

Reply:
@khanhtc1202 Addressed it!